cli/src/semgrep/rule_match.py

```diff
@@ -513,6 +554,28 @@ def to_app_finding_format(self, commit_date: str) -> out.Finding:
             ret.sca_info = self.extra["sca_info"]
         return ret
 
+    @property
+    def scan_source(self) -> RuleScanSource:
```

Comment:
This should be autogenerated from ATD definitions. This boilerplate is annoying to maintain manually.

Reply:
As discussed, I am adding a #todo here so we can tackle it as a part of strongly typed interfaces work that we plan to do soon!